pkg/zsio/detector/lookup.go

```diff
@@ -0,0 +1,60 @@
+package detector
```

Comment:
Is this move needed for the utf changes of this PR?

Reply:
It was a change I wanted to make for a while (I prefer package references going up in the tree not down) and I needed it to put Flags in zsio without an import cycle.